pkgs/top-level/python-packages.nix

```diff
@@ -13940,6 +13940,23 @@ in {
     };
   });
 
+  ltc_scrypt = buildPythonPackage rec {
+    pname = "ltc_scrypt";
+    version = "1.0";
+    name = "${pname}-${version}";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "1h90hh3iw4i7zs7jgskdjlk8gi97b3v2zqsxdfwdvhrrnhpvv856";
+    };
+
+    meta = {
+      description = "Bindings for scrypt proof of work used by Litecoin";
+      homepage = https://pypi.python.org/pypi/ltc_scrypt;
+      maintainers = with maintainers; [ asymmetric ];
```

Comment:
actually, looking at the source, it's taken from cperciva's tarsnap, and the license is similar to [this one](https://github.com/Tarsnap/scrypt/blob/master/COPYRIGHT).

Since it's not a standard license, what should I do?